src/main/kotlin/no/nav/syfo/varsel/MerOppfolgingVarselDTO.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.syfo.varsel
+
+data class MerOppfolgingVarselDTO(
+    val personIdent: String,
+    val utbetalingId: String,
+    val sykmeldingId: String,
+)
```

Comment:
Vil et varsel høre til en spesifikk utbetaling og en spesifikk sykmelding? Hva er disse id'ene i denne sammenhengen? 😊

Reply:
Jeg tror at sammehengen er sånn at utbetaling triggerer varsel utsending og før utsendingen sjekkes det at det finnes en aktiv sykmelding på en forrespurt person. Her er det en id til den sykmelding.

Reply:
Stemmer det Tanja, var det jeg tenkte. Ikke sikkert det er nødvendig å lagre det, men tenkte det kanskje kunne være nyttig informasjon for enten Lars (f.eks for å enklere se hvor mange gjenstående sykepengedager man har ved utsendingstidspunkt) - eller for oss dersom vi ønsker å grave selv.